tools/constraints-xenial.txt

```diff
@@ -2,6 +2,7 @@ attrs==15.2
 flake8==2.5.4
 pep8==1.7.0
 py==1.4.31
+pyflakes==2.1.1
```

Comment:
Another minor question. Why are you also pinning it for Xenial?

Reply:
We are pinning for Xenial because it also breaks when running on a focal system because stock xenial version of pytest pulled in is version 1.0.0  seen when you run tox -r -e pyflakes-xenial locally without this change on a focal system
```
$ tox -r -e flake8-xenial | grep pyflakes==
flake8-xenial installed: appdirs==1.4.3,behave==1.2.6,CacheControl==0.12.6,certifi==2019.11.28,chardet==3.0.4,colorama==0.4.3,contextlib2==0.6.0,coverage==5.1,distlib==0.3.0,distro==1.4.0,flake8==2.5.4,html5lib==1.0.1,idna==2.8,lockfile==0.12.2,mccabe==0.4.0,mock==4.0.2,msgpack==0.6.2,packaging==20.3,parse==1.15.0,parse-type==0.5.2,pep517==0.8.2,pep8==1.7.0,pkg-resources==0.0.0,progress==1.5,py==1.4.31,pycodestyle==2.5.0,pyflakes==1.0.0,PyHamcrest==2.0.2,pyparsing==2.4.6,pytest==2.8.7,pytest-cov==2.2.1,pytoml==0.1.21,PyYAML==3.11,requests==2.22.0,retrying==1.3.3,six==1.14.0,ubuntu-advantage-tools==20.3,urllib3==1.25.8,webencodings==0.5.1
```

Reply:
ah ok thanks